integration/build/build_test.go

```diff
@@ -488,15 +488,22 @@ RUN [ ! -f foo ]
 }
 
 // #37581
+// #40444 (Windows Containers only)
 func TestBuildWithHugeFile(t *testing.T) {
-	skip.If(t, testEnv.OSType == "windows")
 	ctx := context.TODO()
 	defer setupTest(t)()
 
 	dockerfile := `FROM busybox
-# create a sparse file with size over 8GB
+`
+
+	if testEnv.DaemonInfo.OSType == "windows" {
+		dockerfile += `# create a file with size over 8GB
```

Comment:
`fsutil.exe file createnew bigfile.txt 8589934592` might work better here.

Reply:
Oh, nice! That creates a sparse file by default? Definitely much better. I'll have a play tonight, that would let me revert the timeout change too, as long as it correctly reproduces the issue.

Reply:
I just confirmed elsewhere, that cut the time for the test using `ue4-docker` in half. So I'll rework this PR to use this in the integration test instead.

Thank you @marosset.

Reply:
Actually I believe fsutil.exe file createnew will just create an empty file. You can follow up with `fsutil.exe sparse setflag bigfile.txt` and `fsutil.exe sparse setrange bigfile.txt 0 1000` to populate some of the bits.
Should still be a lot faster than powershell tho :)
https://www.geeksforgeeks.org/creating-sparse-files-on-windows-linux/

Reply:
An empty file with a large size is file, the bug relates to tar headers for the filesize, rather than anything about the content or in-container storage.

Reply:
That took 3 minutes, instead of 10. Much better, thank you.